static/src/stylesheets/module/commercial/glabs/_hosted-article.scss

```diff
@@ -32,6 +32,8 @@
             }
 
             @include mq(desktop) {
+                // IE 10 and 11 do not recognize 'unset' so will fallback to 'auto' which luckily gives us the desired layout
+                left: auto;
```

Comment:
Does `auto` work across the board? If so, maybe we can lose the `unset` rule altogether?